doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD not process Autocrypt headers
+larger than 10 KiB in size.
+
+.. note::
+
+  - An Autocrypt header following this specification
+    will be up to 3 KiB large.
+    Some users import 4096-bit RSA key configurations,
+    which may result in Autocrypt headers of up to 5 KiB.
```

Comment:
a 4096-bit RSA primary key with a single user id and two 4096-bit subkeys (one for signing, one for encryption), base64-encoded, comes to about 5256 octets plus the length of the base64-encoded user ID.  That's slightly more than 5KiB, and it doesn't even include the header name or other autocrypt attributes.  maybe we want to say "about 5KiB" or "less than 6KiB"?

Reply:
I checked with my own AC header - pasted it to a file an ran wc on it - which resulted in approx 3200 chars.
I'll try to figure out what's going on there.

Reply:
```
$ ls -l azul.b64 
-rw-r--r-- 1 azul azul 3054 Jan 17 08:42 azul.b64
$ wc azul.b64 
  42   42 3054 azul.b64
$ base64 -d azul.b64 | gpg --list-packets
# off=0 ctb=99 tag=6 hlen=3 plen=525
:public key packet:
	version 4, algo 1, created 1324285022, expires 0
	pkey[0]: [4096 bits]
	pkey[1]: [17 bits]
	keyid: 786085AC20D3457D
# off=528 ctb=b4 tag=13 hlen=2 plen=23
:user ID packet: "Azul. <azul@riseup.net>"
# off=553 ctb=89 tag=2 hlen=3 plen=599
:signature packet: algo 1, keyid 786085AC20D3457D
	version 4, created 1538491089, md5len 0, sigclass 0x13
	digest algo 10, begin of digest 16 08
	hashed subpkt 27 len 1 (key flags: 23)
	hashed subpkt 30 len 1 (features: 01)
	hashed subpkt 23 len 1 (keyserver preferences: 80)
	hashed subpkt 25 len 1 (primary user ID)
	hashed subpkt 11 len 4 (pref-sym-algos: 9 8 7 3)
	hashed subpkt 21 len 4 (pref-hash-algos: 10 9 8 11)
	hashed subpkt 22 len 4 (pref-zip-algos: 2 3 1 0)
	hashed subpkt 33 len 21 (issuer fpr v4 BB70B0160449010A00AF49E8786085AC20D3457D)
	hashed subpkt 2 len 4 (sig created 2018-10-02)
	hashed subpkt 9 len 4 (key expires after 7y289d5h41m)
	subpkt 16 len 8 (issuer key ID 786085AC20D3457D)
	data: [4095 bits]
# off=1155 ctb=b9 tag=14 hlen=3 plen=525
:public sub key packet:
	version 4, algo 1, created 1509612037, expires 0
	pkey[0]: [4096 bits]
	pkey[1]: [17 bits]
	keyid: 566E95B3ECCE1EE8
# off=1683 ctb=89 tag=2 hlen=3 plen=572
:signature packet: algo 1, keyid 786085AC20D3457D
	version 4, created 1538491141, md5len 0, sigclass 0x18
	digest algo 10, begin of digest e2 e1
	hashed subpkt 27 len 1 (key flags: 0C)
	hashed subpkt 33 len 21 (issuer fpr v4 BB70B0160449010A00AF49E8786085AC20D3457D)
	hashed subpkt 2 len 4 (sig created 2018-10-02)
	hashed subpkt 9 len 4 (key expires after 1y334d5h58m)
	subpkt 16 len 8 (issuer key ID 786085AC20D3457D)
	data: [4091 bits]
```

Reply:
Your key doesn't have independent signing and certification keys. This is what Autocrypt recommends, but for an "upper bound" we'll have to consider the 3 x RSA 4096 configuration as dkg says

Reply:
yep, i think @Valodim has identified the disparity -- it's not so much *separate* as "a signing-capable subkey" (since the primary could be signing+certification capable, and a signing-capable subkey could also be present).